### DISCUSSION TITLE
Opensource projects

### DISCUSSION THREAD

**Discussion by IMPranshu** [smartcontractkit/full-blockchain-solidity-course-py#976] [Q&A] [answered]
Can anyone suggest some opensource project to contribute to a good first issue??? 
This would certainly help everyone who is just starting to learn blockchain

**Comment by cromewar** [comment, marked as answer]
Hello @IMPranshu Is hard to say, but I strongly suggest you to find into [Chainlink Discord Server](https://discord.com/invite/aSK4zew) there are always people looking for help on their project and most of the are open source as well.
